linter.py

```diff
@@ -64,6 +77,11 @@ def run(self, cmd, code):
             code = ' '
 
         self.ensure_plugin_installed()
+
+        stdin_filename = self.get_stdin_filename()
+        if stdin_filename:
+            cmd.append('--stdin-filename=' + stdin_filename)
+
```

Comment:
This is generally fine-code *but* `cmd` must not be mutated in `run` anymore.  Actually SublimeLinter accepts `cmd` to be a method if the command must be constructed dynamically, like in this case here.  

So remove the `cmd = 'eslint --format json --stdin'` line from the beginning of the file and implement

```
def cmd(self):
	cmd = ['eslint', '--format=json', '--stdin']
	...  # what you did here
	return cmd
```